src/main/drivers/vtx_smartaudio.c

```diff
@@ -0,0 +1,706 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#ifdef VTX_SMARTAUDIO
+
+#include "drivers/system.h"
+#include "drivers/serial.h"
+#include "drivers/vtx_smartaudio.h"
+#include "io/serial.h"
+#include "fc/runtime_config.h"
+#include "config/config_master.h"
```

Comment:
I'm beginning to think about moving this to io since it never touches raw I/O directly, or separating it into a driver that handles up to transport layer (framing + retransmission) and a module in io that handles the vtx control protocol. Thoughts?